databases/dummy_users_data.sql

```diff
@@ -0,0 +1,25 @@
+use devices;
+
+INSERT INTO Users (comments) VALUES ('someone registered with Google OAuth');
```

Comment:
This is probably handled already by the identity provider (Azure B2C in our case, probably)